python/cudaq/kernel/ast_bridge.py

```diff
@@ -246,11 +246,26 @@ def getFloatAttr(self, type, value):
     def getConstantFloat(self, value):
         """
         Create a constant float operation and return its MLIR result Value.
-        Takes as input the concrete float value. 
+        Takes as input the concrete float value.
         """
         ty = self.getFloatType()
         return arith.ConstantOp(ty, self.getFloatAttr(ty, value)).result
 
+    def getComplexType(self):
```

Comment:
It would be good to parameterize this with the precision or bitwidth in MLIR terms (32 or 64 is what we support)

Reply:
Done.